db/warehouse/migrate/20240221195839_migrate_auto_exit_rules.rb

```diff
@@ -0,0 +1,36 @@
+#  Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+#  License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+#
+
+class MigrateAutoExitRules < ActiveRecord::Migration[6.1]
+  def up
+    Hmis::AutoExitConfig.all.each do |old_auto_exit_config|
+      Hmis::ProjectAutoExitConfig.create(
+        project_type: old_auto_exit_config.project_type,
+        organization_id: old_auto_exit_config.organization_id,
+        project_id: old_auto_exit_config.project_id,
+        config_options: { length_of_absence_days: old_auto_exit_config.length_of_absence_days }.to_json,
+        created_at: old_auto_exit_config.created_at,
+        updated_at: old_auto_exit_config.updated_at,
+      )
```

Comment:
Isn't there an "active" or "enabled" status that we need to set here? The old table didn't have that, so we should migrate all old records as active.

Reply:
Yes good catch, thank you!